pyqtgraph/functions.py

```diff
@@ -293,75 +345,115 @@ def mkColor(*args):
     return QtGui.QColor(*args)
 
 
-def mkBrush(*args, **kwds):
+def mkBrush(*args, **kargs):
     """
     | Convenience function for constructing Brush.
     | This function always constructs a solid brush and accepts the same arguments as :func:`mkColor() <pyqtgraph.mkColor>`
     | Calling mkBrush(None) returns an invisible brush.
     """
-    if 'color' in kwds:
-        color = kwds['color']
-    elif len(args) == 1:
-        arg = args[0]
-        if arg is None:
-            return QtGui.QBrush(QtCore.Qt.NoBrush)
-        elif isinstance(arg, QtGui.QBrush):
-            return QtGui.QBrush(arg)
+    while ( # unravel single element sublists
+        isinstance(args, (tuple, list) )
+        and len(args) == 1 
+    ): 
+        args = args[0]
+    # now args is either a non-list entity, or a multi-element tuple
+    # short-circuits:
+    if isinstance(args, NamedBrush):
```

Comment:
no code change requested here, just thinking of how nice the pattern matching functionality in python 3.10 will be able to serve us, ...too bad we won't get access to it for another ~4 years in this library.